testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1032,7 +1032,7 @@
 Then(/^The metadata buildtime from package "(.*?)" match the one in the rpm on "(.*?)"$/) do |pkg, host|
   # for testing buildtime of generated metadata - See bsc#1078056
   node = get_target(host)
-  cmd = "dumpsolv /var/cache/zypp/solv/spacewalk\:test-channel-x86_64/solv | grep -E 'solvable:name|solvable:buildtime'| grep -A1 '#{pkg}$'| perl -ne 'if($_ =~ /^solvable:buildtime:\\s*(\\d+)/) { print $1; }'"
+  cmd = "dumpsolv /var/cache/zypp/solv/spacewalk\:test-channel-for-sle/solv | grep -E 'solvable:name|solvable:buildtime'| grep -A1 '#{pkg}$'| perl -ne 'if($_ =~ /^solvable:buildtime:\\s*(\\d+)/) { print $1; }'"
```

Comment:
That might need to be adapted to the architecture as the other TODOS.

Reply:
That was also exclusively used by the trad client, should we still keep this step?